Apps/W1/HybridGP/app/src/pages/GPCompanyAdditionalSettingsList.Page.al

```diff
@@ -0,0 +1,114 @@
+page 4051 "GP Company Add. Settings List"
+{
+    Caption = 'GP Company Additional Settings List';
+    PageType = ListPart;
+    SourceTable = "GP Company Additional Settings";
+    SourceTableView = sorting(Name) where("Name" = filter(<> ''));
+    DeleteAllowed = false;
+    InsertAllowed = false;
+    ModifyAllowed = true;
+
+    layout
+    {
+        area(content)
+        {
+            repeater(General)
+            {
+                field(Name; Rec.Name)
+                {
+                    Caption = 'Company';
+                    ToolTip = 'Specifies the value of the Name field.';
```

Comment:
Should 'Company' be used somewhere in this tooltip?

Reply:
Ha! Yes, I need to update this.